src/Microsoft.Health.Dicom.Core/Features/Store/IStoreOrchestrator.cs

```diff
@@ -18,10 +19,14 @@ public interface IStoreOrchestrator
     /// Asynchronously orchestrate the storing of a DICOM instance entry.
     /// </summary>
     /// <param name="dicomInstanceEntry">The DICOM instance entry to store.</param>
+    /// <param name="dicomDataset">Dataset to use to store JSON metadata</param>
```

Comment:
See my comment where this is called in `StoreService` - we shouldn't need this parameter.